src/hb-subset-input.hh

```diff
@@ -62,7 +62,7 @@ struct hb_subset_input_t
   bool attach_accelerator_data = false;
   hb_hashmap_t<hb_tag_t, float> *axes_location;
 #ifdef HB_EXPERIMENTAL_API
-  hb_hashmap_t<unsigned, hb_bytes_t> *name_table_overrides;
+  hb_hashmap_t<hb_vector_t<unsigned>, hb_bytes_t> *name_table_overrides;
```

Comment:
Suggest having a custom struct to hold the id values.

Reply:
Done.